dnf-behave-tests/features/obsoletes.feature

```diff
@@ -31,6 +31,40 @@ Scenario: Install of obsoleting package, even though higher version than obsolet
         | install       | PackageA-Obsoleter-0:1.0-1.x86_64         |
 
 
+Scenario: Do not install of obsoleting package, when obsoleted package not on the system
+   When I execute dnf with args "upgrade PackageA-Obsoleter"
+   Then the exit code is 1
+    And Transaction is empty
+
+
+@bz1818118
+Scenario: Install of obsoleting package, when obsoleted package on the system
```

Comment:
Could you add to all the scenario titles that the install is done using the `upgrade` command? 
I think it would make it more obvious what is being tested here.
Maybe something like:
```Gherkin
Scenario: Install of obsoleting package via the upgrade command, when obsoleted package on the system
```